### DISCUSSION TITLE
Table selection

### DISCUSSION THREAD

**Discussion by insomniacj93** [Budibase/budibase#6454] [Help] [answered]
Hello there,

I was hoping for a way to restrict the row selection to just 1 (de-selecting the previous row automaticly). 

Before this I was trying to transfer a bulk of rows from one table to another, tried the Continue if Stop if but without success.

Is there any way to still try to make the Multi row transfer work, perhaps a loop automation?

**Comment by melohagan** [comment, marked as answer]
Hello @insomniacj93 

1. No way to restrict the row selection within a table. You may be able to use Repeaters and the Checkbox component to build that functionality.
2. You should be able to use the **Query Rows** action to get rows that can be looped over to trigger a **Create Row** action in the destination table.
